src/xrCore/LocatorAPI_defs.cpp

```diff
@@ -105,6 +105,17 @@ LPCSTR FS_Path::_update(string_path& dest, LPCSTR src) const
     R_ASSERT(src);
     string_path temp;
     xr_strcpy(temp, sizeof(temp), src);
+
+#ifdef XR_PLATFORM_LINUX
+    string_path fullPath;
+    strconcat(sizeof(fullPath), fullPath, m_Path, temp);
+    xr_fs_strlwr(fullPath);
+    if (FS.exist(fullPath, FSType::External) || FS.exist(fullPath, FSType::Virtual)) {
+        xr_strcpy(dest, sizeof(dest), fullPath);
```

Comment:
sizeof(dest) not needed
```suggestion
        xr_strcpy(dest, fullPath);
```

Reply:
Done